tests/python/python-peewee-mysql-hardcoded-secret-python-test.yml

```
@@ -4,4 +4,4 @@ valid:
     mysql_db1 = MySQLDatabe('my_app', user='app', password=os.env['password'], host='10.1.0.8', port=3306)
 invalid:
   - |
-    mysql_db1 = MySQLDatabase('my_app', user='app', password='db_password', host='10.1.0.8', port=3306) 
\ No newline at end of file
+    mysql_db1 = MySQLDatabase('my_app', user='app', password='db_password', host='10.1.0.8', port=3306) 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)